packages/network/test/integration/message-propagation.test.ts

```diff
@@ -96,24 +95,26 @@ describe('message propagation in network', () => {
             payload: streamMessage.getParsedContent()
         }))
 
+        n1.subscribe('stream-1', 0)
```

Comment:
I wonder whether `subscribe` should return a `waitForNeighbors` promise, maybe as `asyncSubscribe`

Reply:
I don't see this necessarily being a very common use case compared to e.g. publishing? Usually you'd just want to subscribe to a stream and then assign a handler for "when the messages start eventually appearing". For those situations where you want to wait, you can use `waitForNeighbors`?